_posts/2017-03-16-two-months-of-tech-interviews.md

```diff
@@ -0,0 +1,220 @@
+- **Team Lunch:** Most companies with full-day onsites will schedule in some sort of lunch interview. It's supposed to be a chance to relax and ask questions to an engineer. At one company, the lunch was with my entire prospective team. Having everyone meant I could just shut up and see how people interacted regularly. I also got to feel what it was like to spend time with my potential future colleagues.
+
+Besides the interviews themselves, I also had some really good recruiters. People who were super attentive and eager to move things along at whatever pace I was comfortable.
+
+Interviewing was stressful, so every supportive recruiter and well thought out interview was much appreciated ❤️.
+
+### Offers
+
+Learning I had offers was really stressful. 
+
+> Do I take it or finish interviewing with these other companies?
+
+> Is this my market value? Should I negotiate?
+
+> Company A is really cool, but I'll have to move so far. Company B is smaller, but in a better city. How do I choose??
+
+I don't plan on moving again for a while, so it was a big decision. The factors that led me to choose Squarespace:
+
+- Well established company with a good reputation.
+- Team with a bunch of friendly people and interesting projects.
+- Being close to the Recurse Center and the awesome communities in New York.
+- At the same time, being close enough to Toronto that I can visit easily.
+
+Places in Vancouver were a close second. I met a bunch of wonderful people there during my time at Mobify. Still sad that I won't be as close to them as I would like 😢.
+
+### Some notes for past me
+
+Now that it's all over, what tips would I have for past me?
+
+- **Prepare early:** Interviews are a whole other skill-set - you can't just wing it.
+- **Clean up your resume:** Squishing a years worth of activities into 1 item is confusing. It's better to break things up properly and skip some details.
+- **Focus on referrals:** It might be easier to upload your resume everywhere, but your chances are way higher with referrals.
+- **Research the companies:** Interviews are much more interesting if you can go in knowing what they're working on and why you want to contribute.
+- **Write more code by hand:** Interviewing involves a ton of communication. How you work through the problem is just as important as coding the final solution. Practice that thought process.
+- **Focus on interviewing resources:** Things like HackerRank are fun because you get to write code and test it out. But working code isn't everything. Resources like "Cracking The Coding Interview" give in depth explanations of all the other things interviewers are looking for. Did you realize there's a faster solution that uses more memory? Or that assuming a sorted input let's you cut the runtime in half?
```

Comment:
Hmmm... I could see both being correct. I was talking about working code as a general concept, whereas `the` would be referring to the code described in the previous sentence.

I think it's fine as is.